src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/Dependencies/ResolvedFrameworkReference.xaml

```diff
@@ -5,7 +5,6 @@
       DisplayName="Framework Reference"
       PageTemplate="generic"
       xmlns="http://schemas.microsoft.com/build/2009/properties">
-  
```

Comment:
This blank line removal is on purpose. It makes the formatting of this file match all others in this folder.